src/ert/scheduler/openpbs_driver.py

```diff
@@ -202,13 +205,24 @@ async def submit(
         /,
         *args: str,
         name: str = "dummy",
-        runpath: Optional[str] = None,
+        runpath: Optional[Path] = None,
     ) -> None:
+        if runpath is None:
+            runpath = Path.cwd()
+
         arg_queue_name = ["-q", self._queue_name] if self._queue_name else []
         arg_keep_qsub_output = (
             [] if self._keep_qsub_output else "-o /dev/null -e /dev/null".split()
         )
 
+        script = (
+            "#!/usr/bin/env bash\n"
+            f"cd {shlex.quote(str(runpath))}\n"
+            f"exec -a {shlex.quote(executable)} {executable} {shlex.join(args)}\n"
+        )
+        script_path = runpath / "job_script.sh"
```

Comment:
Why not to use the same `tempfile.NamedTemporaryFile` since when `runpath=None` this will not work or ?

Reply:
Good catch, it is not used any longer after changing this PR to use stdin